.github/workflows/dockerhub-publish.yml

```diff
@@ -0,0 +1,76 @@
+name: Docker Publish
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+    - cron: "0 3 * * *"
+  workflow_dispatch:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - imageName: projector-webstorm
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/webstorm/WebStorm-2020.3.3.tar.gz
+          - imageName: projector-idea-c
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/idea/ideaIC-2020.3.4.tar.gz
```

Comment:
How can we add one more version here? For example, if we want to also build 2020.1 and make it `latest` – the current logic will cause a clash because `latest` will be pushed two times.

We need to prevent it somehow. For example, we can add an optional `latest: true` property for each matrix entry and execute step that pushes `latest` tag only if this property is set and it's set to true. Maybe there is a better solution. Wdyt?

Reply:
My initial logic is that we will push the latest versions of images first (not `latest` tag, just latest versions). From now on, we will only push the next versions and update the `latest` tag.

For example, when initializing the repository, we will push the versions of all IDE 2020.3.4. Now we have `latest` tag and `2020.3.4` tag. Under `latest` tag is used `2020.3.4`. Now we want to push version 2020.3.5, we should update all tags and links on the script, and after pushing in the repository we have 3 tags: `latest`(2020.3.5), `2020.3.4`, `2020.3.5`

Reply:
Am I right that after we add `2020.3.5`, we won't update `2020.3.4` anymore?